datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Domain/SidebarDomainSection.tsx

```diff
@@ -53,31 +74,34 @@ export const SidebarDomainSection = ({ readOnly }: Props) => {
         <div>
             <div id={ENTITY_PROFILE_DOMAINS_ID} className="sidebar-domain-section">
                 <SidebarHeader title="Domain" />
-                <div>
+                <ContentWrapper displayInline={!!domain}>
```

Comment:
I think I may be missing why we need these changes!

Reply:
yeah the main reason for these changes are that we don't want people to be able to delete a domain off of their data product, but we do want to allow them to change it if they want. So data product profile pages will have the domain name and the "set domain" button next to it. This is just CSS updates to make it look nicer

Reply:
ahhh! okay got it